test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,582 @@
+package functional
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+
+	"github.com/integr8ly/cloud-resource-operator/pkg/resources"
+
+	"github.com/aws/aws-sdk-go/service/elasticache"
+
+	v1 "k8s.io/api/core/v1"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/aws-sdk-go/service/rds"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/integr8ly/integreatly-operator/test/common"
+)
+
+var (
+	resourceType    = "_network"
+	tier            = "development"
```

Comment:
Is the `development` tier valid for BYOC clusters as well?

Reply:
No good catch, this should be changed!